R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1478,6 +1481,13 @@ test_that("column functions", {
     lapply(
       list(list(x = 1, y = -1, z = -2), list(x = 2, y = 3,  z = 5)),
       as.environment))
+
+  df <- as.DataFrame(data.frame(is_true = c(TRUE, FALSE, NA)))
+  expect_equal(
+    collect(select(df, alias(SparkR::not(df$is_true), "is_false"))),
```

Comment:
Initially I experienced some shading issues with `testthat::not`, but I cannot reproduce this any longer. Removed.